_posts/2021-01-13-oklab-critique.md

```diff
@@ -272,6 +275,24 @@ This blog post benefitted greatly from conversations with Björn Ottson and Jaco
 
 Discuss on [Hacker News](https://news.ycombinator.com/item?id=25830327).
 
+## Update 2021-01-29
+
+I've added several more color spaces to the widget above. You can get to them by clicking the last gradient. (I'd happily accept a patch to make this more general; this blog is open source).
```

Comment:
> You can get to them by clicking the last gradient.

I read this and couldn’t figure out what to do. I was clicking on the last *button* to change the gradient. I had to look at this patch to figure out what to do.

Reply:
Submitted https://github.com/raphlinus/raphlinus.github.io/pull/45 to do it a different way that I reckon better.